lib/storage/index.js

```diff
@@ -0,0 +1,266 @@
+import _ from 'underscore';
+import PlatformStorage from './platforms';
+import MemoryOnlyProvider from './providers/MemoryOnlyProvider';
+import InstanceSync from './InstanceSync';
+import * as Logger from '../Logger';
+
+let storageProviders = [PlatformStorage, MemoryOnlyProvider];
+let finishInitalization;
+const initPromise = new Promise((resolve) => {
+    finishInitalization = resolve;
+});
+let shouldKeepInstancesSync = false;
+
+/**
+ * Runs an operation consecutively on all storage providers
```

Comment:
**To improve this comment answer these questions:**

What is an  example of an "operation"? 
Why it must run on "all storage providers"? 
Why do we want to run it "consecutively"?

Reply:
i updated the comments of these helper functions